wlog.go

```diff
@@ -142,7 +152,7 @@ func (l *Logger) WithScope(fields Fields) *ScopedLogger {
 	for k, v := range l.fields {
 		scopeFields[k] = v
 	}
-	return &ScopedLogger{scopeFields, l, l.formatter}
+	return &ScopedLogger{scopeFields, l, l.formatter, sync.Mutex{}}
```

Comment:
sync.Mutex needs no initialization. It's redundant. Please remove

Reply:
Fixed